pkg/scheduler/api/compatibility/compatibility_test.go

```diff
@@ -1223,7 +1301,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 			)
 
 			if err != nil {
-				t.Fatalf("%s: Error constructing: %v", v, err)
+				t.Fatalf("%s: Error constructing: %v", tc.name, err)
```

Comment:
It's ok to not include `tc.name` here.

```suggestion
				t.Fatalf("Error constructing: %v", err)
```

In each `t.Run()`, we've already put `tc.name`, so Golang will append `/<tc.name>` to the testcase name in the output.